packages/babel-plugin-syntax-pipeline-operator/README.md

```diff
@@ -0,0 +1,35 @@
+# babel-plugin-syntax-pipeline-operator
+
+Allow parsing of pipeline operator `|>`. See [the ES.next proposal](https://github.com/mindeavor/es-pipeline-operator) for details.
```

Comment:
`of` -> `of the`? to be consistent with `Allow parsing of the pipeline operator` in the package.json description?